base/src/main/java/org/aya/core/def/PrimDef.java

```diff
@@ -246,8 +205,6 @@ public void clear(@NotNull ID name) {
   public enum ID {
     /** Short for <em>Arend coe</em>. */
     ARCOE("arcoe"),
-    SQUEEZE_LEFT("squeezeL"),
```

Comment:
I think these functions can still be primitives, so they can be renamed. We can transform them into special ASTs in elaboration and leave the reduction function here as the identity function